bucket/linqpad5.json

```diff
@@ -0,0 +1,24 @@
+{
+    "version": "5.43.0",
+    "homepage": "https://www.linqpad.net",
+    "description": "LINQPad 5 for .NET Framework 4.6/4.7/4.8",
+    "license": {
+        "identifier": "Freeware",
+        "url": "https://www.linqpad.net/eula.txt"
+    },
+    "url": "https://linqpad.azureedge.net/public/LINQPad5.zip?cache=5.43.0.8121888",
+    "hash": "7178cb0165963841fffbd0fce6a3f8d6c00571cb99622f1e5e472d523b91dd8c",
+    "shortcuts": [
+        [
+            "linqpad.exe",
+            "LINQPad"
+        ]
+    ],
+    "checkver": {
+        "url": "https://www.linqpad.net/Download.aspx",
+        "regex": "LINQPad5Version\\"\\>([\\d.]+)\\<"
```

Comment:
should be `"regex": "LINQPad5Version\\\"\\>([\\d.]+)\\<"`
`\\` `\"` in json -> `\"` when loaded into the program

Reply:
Oh, it seems I have deleted my fork and the latest version is now 5.44.02. Maybe you can just copy the proposed changes with some edits (I dont car that its not my commit). Or should I make a new PR?

Reply:
Just make a new PR.